tests/pages/password.js

```diff
@@ -0,0 +1,16 @@
+import {
+  create,
+  visitable
+} from 'ember-cli-page-object';
+import form from './components/password/forgot-password';
+import resetPasswordForm from './components/password/reset-password';
+import errorFormatter from './components/error-formatter';
+
+export default create({
+  visitReset: visitable('/password/reset'),
+  visitForgot: visitable('/password/forgot'),
+
+  form,
+  resetPasswordForm,
+  errorFormatter
+});
```

Comment:
I believe a page object should match with a route 1 to 1. Otherwise, we risk confusing people. Is there any particular reason why you chose to merge the two password pages?

Split up, we would not have `visitReset` and `visitForgot`. The two pages would simply have a `visit`, a `form` and an `errorFormatter`.